src/core/mod.rs

```diff
@@ -110,17 +110,15 @@ impl BPFBuilder {
     ))]
     /// Try constructing a BPF module from the builder
     pub fn build(self) -> Result<BPF, BccError> {
-        let cflags_ptr = if self.cflags.is_empty() {
-            ptr::null_mut()
-        } else {
-            self.cflags.as_ptr() as *mut *const c_char
-        };
-
         let ptr = unsafe {
             bpf_module_create_c_from_string(
                 self.code.as_ptr(),
                 2,
-                cflags_ptr,
+                self.cflags
```

Comment:
This should be ok, even if the vector is empty, as the size will be 0 and bcc won't run any iteration. 

BTW this call inline is a bit messier, but this is the safest (no `std::mem::forget`) version I could come up with